### ISSUE TITLE
AT next zlib is not building for cross compile

### ISSUE THREAD

**Issue by lucmaga** [advancetoolchain/advance-toolchain#1405]
Failure log
```
make[1]: Entering directory '/home/lamm/projects/advance-toolchain/at14.0-0-alpha.ubuntu-18_x86_64_ppc64le/builds/zlib_1-64'
/home/lamm/projects/advance-toolchain/work/at-next-14.0-0-alpha/bin/powerpc64le-linux-gnu-gcc -m64 -g -O3 -mcpu=power8 -D_LARGEFILE64_SOURCE=1 -DHAVE_HIDDEN -DHAVE_IFUNC -DZ_POWER8 -DZ_POWER9 -DZ_POWER_OPT -I. -I/home/lamm/projects/advance-toolchain/at14.0-0-alpha.ubuntu-18_x86_64_ppc64le/sources/zlib/ -c -o adler32_test.o /home/lamm/projects/advance-toolchain/at14.0-0-alpha.ubuntu-18_x86_64_ppc64le/sources/zlib/test/adler32_test.c
/home/lamm/projects/advance-toolchain/work/at-next-14.0-0-alpha/bin/powerpc64le-linux-gnu-gcc -m64 -g -O3 -mcpu=power8 -D_LARGEFILE64_SOURCE=1 -DHAVE_HIDDEN -DHAVE_IFUNC -DZ_POWER8 -DZ_POWER9 -DZ_POWER_OPT -include zconf.h -c -o adler32.o /home/lamm/projects/advance-toolchain/at14.0-0-alpha.ubuntu-18_x86_64_ppc64le/sources/zlib/adler32.c
/home/lamm/projects/advance-toolchain/work/at-next-14.0-0-alpha/bin/powerpc64le-linux-gnu-gcc -m64 -g -O3 -mcpu=power8 -D_LARGEFILE64_SOURCE=1 -DHAVE_HIDDEN -DHAVE_IFUNC -DZ_POWER8 -DZ_POWER9 -DZ_POWER_OPT -include zconf.h -c -o crc32.o /home/lamm/projects/advance-toolchain/at14.0-0-alpha.ubuntu-18_x86_64_ppc64le/sources/zlib/crc32.c
/home/lamm/projects/advance-toolchain/work/at-next-14.0-0-alpha/bin/powerpc64le-linux-gnu-gcc -m64 -g -O3 -mcpu=power8 -D_LARGEFILE64_SOURCE=1 -DHAVE_HIDDEN -DHAVE_IFUNC -DZ_POWER8 -DZ_POWER9 -DZ_POWER_OPT -include zconf.h -c -o deflate.o /home/lamm/projects/advance-toolchain/at14.0-0-alpha.ubuntu-18_x86_64_ppc64le/sources/zlib/deflate.c
/home/lamm/projects/advance-toolchain/work/at-next-14.0-0-alpha/bin/powerpc64le-linux-gnu-gcc -m64 -g -O3 -mcpu=power8 -D_LARGEFILE64_SOURCE=1 -DHAVE_HIDDEN -DHAVE_IFUNC -DZ_POWER8 -DZ_POWER9 -DZ_POWER_OPT -include zconf.h -c -o infback.o /home/lamm/projects/advance-toolchain/at14.0-0-alpha.ubuntu-18_x86_64_ppc64le/sources/zlib/infback.c
/home/lamm/projects/advance-toolchain/work/at-next-14.0-0-alpha/bin/powerpc64le-linux-gnu-gcc -m64 -g -O3 -mcpu=power8 -D_LARGEFILE64_SOURCE=1 -DHAVE_HIDDEN -DHAVE_IFUNC -DZ_POWER8 -DZ_POWER9 -DZ_POWER_OPT -include zconf.h -c -o inffast.o /home/lamm/projects/advance-toolchain/at14.0-0-alpha.ubuntu-18_x86_64_ppc64le/sources/zlib/inffast.c
/home/lamm/projects/advance-toolchain/work/at-next-14.0-0-alpha/bin/powerpc64le-linux-gnu-gcc -m64 -g -O3 -mcpu=power8 -D_LARGEFILE64_SOURCE=1 -DHAVE_HIDDEN -DHAVE_IFUNC -DZ_POWER8 -DZ_POWER9 -DZ_POWER_OPT -include zconf.h -c -o inflate.o /home/lamm/projects/advance-toolchain/at14.0-0-alpha.ubuntu-18_x86_64_ppc64le/sources/zlib/inflate.c
/home/lamm/projects/advance-toolchain/work/at-next-14.0-0-alpha/bin/powerpc64le-linux-gnu-gcc -m64 -g -O3 -mcpu=power8 -D_LARGEFILE64_SOURCE=1 -DHAVE_HIDDEN -DHAVE_IFUNC -DZ_POWER8 -DZ_POWER9 -DZ_POWER_OPT -include zconf.h -c -o inftrees.o /home/lamm/projects/advance-toolchain/at14.0-0-alpha.ubuntu-18_x86_64_ppc64le/sources/zlib/inftrees.c
/home/lamm/projects/advance-toolchain/work/at-next-14.0-0-alpha/bin/powerpc64le-linux-gnu-gcc -m64 -g -O3 -mcpu=power8 -D_LARGEFILE64_SOURCE=1 -DHAVE_HIDDEN -DHAVE_IFUNC -DZ_POWER8 -DZ_POWER9 -DZ_POWER_OPT -include zconf.h -c -o trees.o /home/lamm/projects/advance-toolchain/at14.0-0-alpha.ubuntu-18_x86_64_ppc64le/sources/zlib/trees.c
/home/lamm/projects/advance-toolchain/work/at-next-14.0-0-alpha/bin/powerpc64le-linux-gnu-gcc -m64 -g -O3 -mcpu=power8 -D_LARGEFILE64_SOURCE=1 -DHAVE_HIDDEN -DHAVE_IFUNC -DZ_POWER8 -DZ_POWER9 -DZ_POWER_OPT -include zconf.h -c -o zutil.o /home/lamm/projects/advance-toolchain/at14.0-0-alpha.ubuntu-18_x86_64_ppc64le/sources/zlib/zutil.c
/home/lamm/projects/advance-toolchain/work/at-next-14.0-0-alpha/bin/powerpc64le-linux-gnu-gcc -m64 -g -O3 -mcpu=power8 -D_LARGEFILE64_SOURCE=1 -DHAVE_HIDDEN -DHAVE_IFUNC -DZ_POWER8 -DZ_POWER9 -DZ_POWER_OPT -include zconf.h -c -o compress.o /home/lamm/projects/advance-toolchain/at14.0-0-alpha.ubuntu-18_x86_64_ppc64le/sources/zlib/compress.c
In file included from /home/lamm/projects/advance-toolchain/at14.0-0-alpha.ubuntu-18_x86_64_ppc64le/sources/zlib/contrib/power/slide_hash_resolver.c:6,
                 from /home/lamm/projects/advance-toolchain/at14.0-0-alpha.ubuntu-18_x86_64_ppc64le/sources/zlib/deflate.c:239:
/home/lamm/projects/advance-toolchain/at14.0-0-alpha.ubuntu-18_x86_64_ppc64le/sources/zlib/contrib/power/power.h:7:10: fatal error: zconf.h: No such file or directory
    7 | #include "zconf.h"
      |          ^~~~~~~~~
compilation terminated.
In file included from /home/lamm/projects/advance-toolchain/at14.0-0-alpha.ubuntu-18_x86_64_ppc64le/sources/zlib/contrib/power/adler32_resolver.c:6,
                 from /home/lamm/projects/advance-toolchain/at14.0-0-alpha.ubuntu-18_x86_64_ppc64le/sources/zlib/adler32.c:150:
/home/lamm/projects/advance-toolchain/at14.0-0-alpha.ubuntu-18_x86_64_ppc64le/sources/zlib/contrib/power/power.h:7:10: fatal error: zconf.h: No such file or directory
    7 | #include "zconf.h"
      |          ^~~~~~~~~
compilation terminated.
Makefile:172: recipe for target 'adler32.o' failed
make[1]: *** [adler32.o] Error 1
make[1]: *** Waiting for unfinished jobs....
Makefile:184: recipe for target 'deflate.o' failed
make[1]: *** [deflate.o] Error 1
In file included from /home/lamm/projects/advance-toolchain/at14.0-0-alpha.ubuntu-18_x86_64_ppc64le/sources/zlib/contrib/power/crc32_z_resolver.c:6,
                 from /home/lamm/projects/advance-toolchain/at14.0-0-alpha.ubuntu-18_x86_64_ppc64le/sources/zlib/crc32.c:246:
/home/lamm/projects/advance-toolchain/at14.0-0-alpha.ubuntu-18_x86_64_ppc64le/sources/zlib/contrib/power/power.h:7:10: fatal error: zconf.h: No such file or directory
    7 | #include "zconf.h"
      |          ^~~~~~~~~
compilation terminated.
Makefile:178: recipe for target 'crc32.o' failed
make[1]: *** [crc32.o] Error 1
make[1]: Leaving directory '/home/lamm/projects/advance-toolchain/at14.0-0-alpha.ubuntu-18_x86_64_ppc64le/builds/zlib_1-64'
```

This can be fixed by removing the `#include "zconfig.h"` line. As seen on the log the whole code is already compiled with `-include zconfig.h` so there is no need for this include.

**Comment by lucmaga** [comment]
[logs.tar.gz](https://github.com/advancetoolchain/advance-toolchain/files/4531301/logs.tar.gz)
[_zlib_1-64-3_standard_buildf-06_make.log](https://github.com/advancetoolchain/advance-toolchain/files/4531303/_zlib_1-64-3_standard_buildf-06_make.log)

**Comment by lucmaga** [comment]
Closed with #1417
